Core/Core/FilePicker/FilePickerViewController.swift

```diff
@@ -276,6 +290,10 @@ extension FilePickerViewController: UIImagePickerControllerDelegate, UINavigatio
 }
 
 extension FilePickerViewController: UITableViewDelegate, UITableViewDataSource {
+    public func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
```

Comment:
is this method even needed if there is only `1`?

Reply:
Not needed. Removed. Thanks.